proto/encore/parser/meta/v1/meta.proto

```diff
@@ -45,6 +45,7 @@ message Service {
   repeated RPC         rpcs       = 3;
   repeated DBMigration migrations = 4;
   repeated string      databases  = 5; // databases this service connects to
+  bool                 has_config = 6; // true if the service has uses config
```

Comment:
are you going to use this in the deploy step?

Reply:
Yeah it's being used to detect which services need config computed for them